pkg/externaloidc/oidc/externalclaims/resolver/resolver_test.go

```
@@ -0,0 +1,417 @@
+	mux.HandleFunc("/claims", func(w http.ResponseWriter, r *http.Request) {
+		if r.Header.Get("Authorization") != fmt.Sprintf("Bearer %s", validToken) {
+			w.WriteHeader(http.StatusUnauthorized)
+			return
+		}
+
+		out, err := json.Marshal(responseClaims)
+		if err != nil {
+			panic("couldn't marshal response claims")
+		}
+
+		w.Write(out)
```

golangci-lint (2.11.4)

[error] Error return value of `w.Write` is not checked

(errcheck)